frontend/src/components/metadata/CreateMetadata.tsx

```diff
@@ -38,11 +38,20 @@ export const CreateMetadata = (props: MetadataType) => {
 			{metadataDefinitionList.map((metadata, idx) => {
 				return (
 					<Box className="inputGroup" key={idx}>
+						{sourceItem === "datasets" && metadata.required_for_items.datasets ? (
+						  <Typography>
+							This metadata is required for creating new datasets.
+						  </Typography>
+						) : sourceItem === "files" && metadata.required_for_items.files ? (
+						  <Typography>
+							This metadata is required for creating new files.
```

Comment:
<img width="1246" alt="image" src="https://github.com/clowder-framework/clowder2/assets/13950475/79d181df-a2da-46ee-be00-a07caeac194e">
Could we style this prompt with different color/font/size so it just stands out a little bit more?
I totally missed this when scan over the dialog.